_scripts/toolbox-run

```diff
@@ -1,3 +1,8 @@
-#!/bin/sh
+#!/bin/bash
 
-toolbox run -c cockpit-website bundle exec jekyll server $@
+if [[ -n $1 ]]
```

Comment:
To me this is a bit unfamiliar and a bashism. The canonical/POSIX way is `if [ -n "$1" ]`. But should be ok.

Reply:
It's definitely a bashism: https://stackoverflow.com/questions/3427872/whats-the-difference-between-and-in-bash

I could change it to what you suggest and switch back to sh though; probably fine here.